Project.toml

```diff
@@ -1,15 +1,15 @@
 name = "UnitfulAstro"
 uuid = "f2a1aa4e-5634-4978-a6f6-8c9e368f14ef"
 license = "MIT"
-version = "0.2.0"
+version = "0.3.0"
 
 [deps]
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 UnitfulAngles = "6fb2a4bd-7999-5318-a3b2-8ad61056cd98"
 
 [compat]
-Unitful = "≥ 0.13.0"
-julia = "≥ 1.0.0"
+Unitful = "^0"
```

Comment:
This line means that _all_ versions of Unitful.jl are compatible, which is wrong. I guess the minimum required version is still 0.13, right?

Reply:
According to the [julia semvar rules](https://julialang.github.io/Pkg.jl/stable/compatibility/), this matches `[0.0.0, 1.0.0)` which I now realize is not what we want. I'll update to `"0.13.0, 1.0.0"`

Reply:
What's wrong with `^0.13.0`?  Also, I don't think the previous `Unitful = "≥ 0.13.0"` was so bad, 99% of the packages out there use `≥` and not `^`.

Reply:
I guess the benefit if `^` over `>=` is that `^` provides an upper bound, which is much safer to change than `>=`. Semvar gets funky for major version < 0, which causes issues/confusion. With `"0.13.0, 1.0.0"` we are saying that we are invariant to minor version changes, but have to re-evaluate when Unitful hits `1.0.0`.

With all that being said, I will revert to the previous for simplicity.

Reply:
Just noticed that the previous version of `Project.toml` made the package non-installable: https://travis-ci.org/JuliaAstro/UnitfulAstro.jl/builds/565651240